src/modules/design/bpmn-diff-view/bpmn-diff-view.ts

```diff
@@ -117,14 +119,20 @@ export class BpmnDiffView {
         this.showChangeList = !this.showChangeList;
       }),
 
-      this._eventAggregator.subscribe(environment.events.diffView.setDiffDestination, async(diffDestination: string) => {
-        this._diffDestination = diffDestination;
+      this._eventAggregator.subscribe(environment.events.diffView.setDiffDestination, async(data: Array<string>) => {
+        [this._diffDestination, this._diagramName] = data;
 
-        const diffLastSavedXml: boolean = diffDestination === 'lastSaved';
+        const diffLastSavedXml: boolean = this._diffDestination === 'lastSaved';
         if (diffLastSavedXml) {
           this._setSavedProcessModelAsPreviousXml();
         } else {
           const updatingDeployedXmlWasSuccessfull: boolean = await this._updateDeployedXml();
```

Comment:
```suggestion
          const updatingDeployedXmlWasSuccessful: boolean = await this._updateDeployedXml();
```
double l only in successfully